spec/features/unauth_users/user_creates_account_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+require "support/factory_girl"
+
+RSpec.feature "Unauthenticated user can create an account", type: :feature do
+  it "can successfully create a new account" do
+		visit login_path
+		click_link_or_button("Create an account here.")
+		fill_in("user[first_name]", with: "Richard")
+		fill_in("user[last_name]", with: "Mejia")
+		fill_in("user[email]", with: "richard@example.com")
+		fill_in("user[password]", with: "password")
+		fill_in("user[password_confirmation]", with: "password")
+		click_link_or_button("Submit")
+		expect(page).to have_content "My Account"
+	end
```

Comment:
`end` at 15, 1 is not aligned with `it "can successfully create a new account" do` at 5, 2<br>Tab detected.